collection_checklist.md

```diff
@@ -28,7 +28,7 @@ Note for reviewers: If you don't know how to check any of the points below, plea
 - [ ] has `README.md`
 - [ ] documentation, examples, and return sections use FQCNs for the `M(..)` [format macros](https://docs.ansible.com/ansible/latest/dev_guide/developing_modules_documenting.html#linking-and-other-format-macros-within-module-documentation) when referring to modules
 - [ ] modules (or plugins) from ansible-core use `ansible.builtin.` as a FQCN prefix
-- [ ] documentation and return sections use `version_added:` containing the *collection* version for which an option, module or plugin was added
+- [ ] documentation and return sections use `version_added:` containing the *collection* version for which an option, module or plugin was added (except cases when they were added in the very first release of the collection)
```

Comment:
I think the original intent was "if `version_added` is there, it must use the collection version (and not the Ansible version, or some other random version)". (See also the RST file, it has a section "When using `version_added` in the documentation:" with the points in there that are summarized here.)

I now see that this point in the checklist can also be interpreted as "`version_added` must always be there".

I think your formulation improves this.